src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int NUMBER_OF_FIGURES = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(10, Color.WHITE);
+    }
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(NUMBER_OF_FIGURES)) {
+            case 0:
+                return new Circle(random.nextInt(100), colorSupplier.getRandomColor());
```

Comment:
for better readability, you can move the creation of each shape into a separate method,
e.g. createCircle()